ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift

```diff
@@ -304,6 +304,12 @@ open class InputViewController: UIInputViewController, KeymanWebDelegate {
       self.swallowBackspaceTextChange = false
       return
     }
+
+    // What the actual heck, Apple.  Apparently, in system-keyboard mode, this is also self-triggered if we erase back to a newline.
```

Comment:
Let's add an issue number to comments like this so we can track more easily when maintaining the code in the future